src/templates/Biopost.vue

```diff
@@ -16,13 +16,13 @@
         />
         <a
           v-if="$page.biopost.contact_links.email"
-          :href="`mailto:${$page.biopost.contact_links.email}`"
+          href="`mailto:${$page.biopost.contact_links.email}`"
```

Comment:
I think here we still want the email to be dynamic, so keep the `:` on the `href`.